phase1/openstack/configure-vm.sh

```diff
@@ -13,6 +13,7 @@ KUBEADM_ENABLE_CLOUD_PROVIDER="${k8s_kubeadm_enable_cloud_provider}"
 KUBEADM_ADVERTISE_ADDRESSES="${k8s_kubeadm_advertise_addresses}"
 KUBEADM_CNI_PLUGIN="${k8s_kubeadm_cni_plugin}"
 KUBEADM_MASTER_IP="${k8s_kubeadm_master_ip}"
+KUBEPROXY_MODE=$(get_metadata "k8s_kubeproxy_mode")
```

Comment:
@Lion-Wei, we do not have get_metatdata function in openstack. please check above lines on how to assign the variable.

Reply:
Sorry for that... : (